helmfile.d/charts/user-alertmanager/templates/alertmanager.yaml

```diff
@@ -25,3 +25,6 @@ spec:
   topologySpreadConstraints:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.image }}
+  image: {{ .Values.image | quote }}
```

Comment:
question: Is there a reason you decided to use quote for the user-alertmanager image twice?

Reply:
The intent is to ensure even an empty string is passed correctly and won't turn into `image:` which may be parsed as `image: null`.